web/static/js/components/surveys/SurveyCard.jsx

```diff
@@ -77,106 +75,337 @@ class SurveyCard extends Component<any> {
+          </a>
         </DropdownItem>
-        {
-          this.movable()
-          ? <DropdownItem>
-            <a onClick={e => this.moveSurvey()}><i className='material-icons'>folder</i>{t('Move to')}</a>
-          </DropdownItem>
-          : null
-        }
-        {
-          this.deletable()
-            ? <DropdownItem>
-              <a onClick={e => this.deleteSurvey()}><i className='material-icons'>delete</i>{t('Delete')}</a>
-            </DropdownItem>
-            : null
-        }
-      </Dropdown>
-    )
+      ))}
+    </Dropdown>
+  )
+}
 
-    const surveyCard = <div className='survey-card'>
-      <Card>
-        <div className='card-content'>
-          <div className='survey-card-status'>
-            <Link className='grey-text' to={redirectTo}>
-              {t('{{percentage}}% of target completed', {percentage: String(Math.round(completionPercentage))})}
-            </Link>
-            { this.actionable() ? actionMenu : null }
-          </div>
-          <div className='card-chart'>
-            <RespondentsChart cumulativePercentages={cumulativePercentages} />
-          </div>
-          <div className='card-status'>
-            <Link className='card-title black-text truncate' title={name} to={redirectTo}>
-              <UntitledIfEmpty text={name} emptyText={untitledSurveyTitle(survey, t)} />
-            </Link>
-            <Link to={redirectTo}>
-              {description}
-              <SurveyStatus survey={survey} short />
-            </Link>
-          </div>
-        </div>
-      </Card>
-    </div>
+ActionMenu.propTypes = {
+  actions: PropTypes.array
+}
+
+class ConfirmationCheckModal extends Component<Props, State> {
+  static propTypes = {
+    modalId: PropTypes.String,
+    t: PropTypes.func
+  }
+
+  constructor(props: Props) {
+    super(props)
+    this.state = {checked: false}
+  }
+
+  open(props) {
+    if (props) this.setState(props)
+    this.refs.modal.open()
+  }
+
+  close() {
+    this.refs.modal.close()
+  }
+
+  toggleChecked() {
+    this.setState((state) => ({ checked: !state.checked }))
+  }
 
-    const inPanelSurveyIndex = !!panelSurveyId
-    const isWaveOfPanelSurvey = panelSurvey || survey.generatesPanelSurvey
-    const showAsPanelSurveyCard = !inPanelSurveyIndex && isWaveOfPanelSurvey
+  onConfirmClick = async (e) => {
+    const { onConfirm } = this.state
+
+    e.preventDefault()
+
+    if (!onConfirm) {
+      this.refs.modal.close()
+      return
+    }
+
+    const res = await onConfirm()
+    res !== false && this.refs.modal.close()
+  }
+
+  render() {
+    const { checked } = this.state
+    const { modalId, t } = this.props
 
     return (
-      <div className='col s12 m6 l4'>
-        {
-          showAsPanelSurveyCard
-          ? <div className='panel-survey-card-0'>
-            <div className='panel-survey-card-1'>
-              <div className='panel-survey-card-2'>
-                { surveyCard }
+      <div style={{ 'text-align': 'center' }}>
```

Comment:
React complains that the `text-align` property doesn't exist:
```suggestion
      <div style={{ textAlign: 'center' }}>
```